tests/tensorflow/nightly/bert-mnli.libsonnet

```diff
@@ -124,5 +138,7 @@ local utils = import 'templates/utils.libsonnet';
     bert + v3_32 + functional + tpuVm + common.RunNightly,
     bert + v2_32 + convergence + tpus.reserved + { schedule: '54 21 * * 1,3,5,6' },
     bert + v3_32 + convergence,
+    bert + v4_8 + functional + tpuVm,
+    bert + v4_32 + functional + tpuVm,
```

Comment:
Can we add convergence configs too? Sorry I missed this

Reply:
Done.